Source/UIKit/UISwitch.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  UISwitch.swift
+//  Rex
+//
+//  Created by David Rodrigues on 07/04/16.
+//  Copyright © 2016 Neil Pankey. All rights reserved.
+//
+
+import ReactiveCocoa
+import UIKit
+
+extension UISwitch {
+
+    /// Wraps a switch's `on` state in a bindable property.
+    public var rex_on: MutableProperty<Bool> {
+
+        let property = associatedProperty(self, key: &onKey, initial: { $0.on }, setter: { $0.on = $1 })
+
+        property <~ rex_controlEvents(.ValueChanged)
+            .filterMap { $0 as? UISwitch }
+            .map { $0.on }
```

Comment:
Chained `*map` operators do look a bit cleaner and make sense semantically. However, signal transformations do have a real cost and can add up perf wise so I like to collapse them. In an ideal world there would be a way that happened automatically. But that's a personal pipe dream.